client/webserver/site/src/js/app.ts

```diff
@@ -576,8 +565,10 @@ export default class Application {
       }
       case 'match': {
         const n = note as MatchNote
-        const ord = this.order(n.orderID)
-        if (ord) updateMatch(ord, n.match)
+        const order = this.order(n.orderID)
+        if (!order) return
```

Comment:
Are we certain that we don't want to pass this note along to the `noteReceivers` anyway?

Reply:
I believe we can, but right now nothing is done by it. Should we send it to the bell, or do you have something in mind for it?

Reply:
> Should we send it to the bell,

I'm not sure what you mean. I'm talking about subscribers to order notes. It is expected that all notes received are passed the `noteReceivers`. I don't see any reason to break that assumption here.

Reply:
@vctt94 Try to push changes in such a way that we can always see what changed in just this PR.  The force-push diff includes a mass of other changes unrelated to this PR.  I can see this `return` has become a `break` if I look at the full PR diff again, but it's much more convenient for each update to have a diff viewable somewhere.

Reply:
sorry for the latest force push, but I am taking now a different approach as suggested by marton.